centos-ci/heketi-functional/run-test.sh

```diff
@@ -90,12 +90,9 @@ else
 	go get github.com/tools/godep
 fi
 
-# time to run the tests!
-cd $GOPATH/src/github.com/heketi/heketi/tests/functional
-
 # need to prevent sudo from disabling the SCL
 # PR: https://github.com/heketi/heketi/pull/395
 grep -q ^_sudo lib.sh || ( curl https://github.com/heketi/heketi/commit/981f84b2f7cf6ea39754a0fa275fdc86eb3affbb.patch | git apply )
 
-scl enable sclo-vagrant1 ./run.sh
+scl enable sclo-vagrant1 make test-functional
```

Comment:
Ok, so to keep the test working with previous versions of heketi, I recommend

~~~shell
if make -q test-functional 2>&1 > /dev/null
then
    scl enable sclo-vagrant1 make test-functional
else
    cd $GOPATH/src/github.com/heketi/heketi/tests/functional
    scl enable sclo-vagrant1 ./run.sh
fi
```

Just a reminder, as you pointed the `make -q` ability out :-)

Reply:
That's pretty much exactly the patch I have applied locally, but I didn't update yet, because tests with `make -q` yielded different results (return codes) on different systems for me... Still investigating.

Reply:
btw, the correct way would be `> dev/null 2>&1`. Otherwise error output will go to the original stdout, afaik.